src/placeos-rest-api/controllers/users.cr

```diff
@@ -173,10 +173,15 @@ module PlaceOS::Api
     ###############################################################################################
 
     protected def find_user
-      id = params["id"]
-      Log.context.set(user_id: id)
-
-      Model::User.find!(id, runopts: {"read_mode" => "majority"})
+      lookup = params["id"]
+      user = if lookup.is_email?
+               Model::User.find_by_emails(authority_id: current_user.authority_id.as(String), emails: [lookup]).to_a.first
```

Comment:
Adding this to correctly trigger the `404` responder. I'd rather not use the explicit ORM `not found` error, but this gets the job done. 
```suggestion
               found = Model::User.find_by_emails(authority_id: current_user.authority_id.as(String), emails: [lookup]).first?
               raise RethinkORM::Error::DocumentNotFound.new if found.nil?
               found
```